Project.toml

```diff
@@ -11,16 +12,17 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 PhysicalConstants = "5ad8b20f-a522-5ce9-bfc9-ddf1d5bda6ab"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SimpleTraits = "699a6c99-e7fa-54fc-8d76-47d257e15c1d"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-julia = "1.6"
 ArgCheck = "2.3.0"
 ConstructionBase = "1.3.0"
-DocStringExtensions = "0.8.5"
+DocStringExtensions = "0.8.5, 0.9"
 PhysicalConstants = "0.2.1"
 SimpleTraits = "0.9.4"
 StaticArrays = "1.2.13"
+julia = "1.6"
```

Comment:
Wasn't this supposed to be increased?

Reply:
We discussed this and decided to put this on the LTS, i.e. `julia="1.6"` (see https://github.com/QEDjl-project/QEDbase.jl/issues/16 for a longer discussion). 

However, since we run the unit tests on `julia=1.6,1.7,1.8,1.9`, we could also add those to the `[compat]`. But I think this is not necessary, because till a new LTS is proclaimed, the newer versions should be back-compatible to the current LTS anyway.